### ISSUE TITLE
Backfilling doesn't work with old prometheuses

### ISSUE THREAD

**Issue by afoninsky** [VictoriaMetrics/VictoriaMetrics#1943]
# Problem
**- old Prometheus versions understand `step` argument in seconds only**
**- VictoriaMetrics converts seconds into human-readable format even if they are explicitly specified**

# Details
When running `vmalert` with backfill enabled against old prometheuses - receive the following error:
```
fatal VictoriaMetrics/app/vmalert/replay.go:96 rule "container_cpu_usage_cpu_cores:namespace": unexpected response code 400 for https://***/api/v1/query_range?end=1631054400&query=sum%28rate%28container_cpu_usage_seconds_total%7Bimage%21%3D%22%22%7D%5B1m%5D%29%29+by+%28cluster%2C+namespace%29&start=1630454400&step=1m0s. Response body {"status":"error","errorType":"bad_data","error":"invalid parameter 'step': cannot parse \"1m0s\" to a valid duration"}
```

When try to specify exact seconds - Victoria automatically converts them to human-readable time so error appears again:
```
groups:
  - name: capacity-current
    interval: 120
    rules:
      ....
```

```
&start=1630454400&step=2m0s. Response body {"status":"error","errorType":"bad_data","error":"invalid parameter 'step': cannot parse \"2m0s\" to a valid duration"}
```

# Possible fix
- pass interval in the form it is specified in config without converting to human-readable format

# Configuration
1. vmalert-20211108-135607-tags-v1.69.0-0-g69e655ba7
```
      - -datasource.url=https://***
      - -datasource.tlsInsecureSkipVerify
      - -remoteWrite.url=http://storage:8428
      - -disableAlertgroupLabel
      - -notifier.url=http://localhost:8880
      - -rule=/etc/config.yaml
      - -replay.timeFrom=2021-09-01T00:00:00Z
      - -replay.timeTo=2021-12-14T23:59:59Z
      - -replay.maxDatapointsPerQuery=10000
      - -replay.ruleRetryAttempts=2
```
2. prometheus v2.17.2 build date 20200420-08:27:08

**Comment by valyala** [comment]
@hagen1778 , could you look at this issue?

**Comment by hagen1778** [comment]
Hi @afoninsky! Could you try to build `vmalert` from https://github.com/VictoriaMetrics/VictoriaMetrics/commit/172d0c5173e87c2f95adaa00030cfc5f3fb23318 commit and try again? See how to [build from sources](https://docs.victoriametrics.com/vmalert.html#how-to-build-from-sources). Thanks!

**Comment by afoninsky** [comment]
Hello @hagen1778 , I do confirm that queries are executed

**Comment by valyala** [comment]
The bugfix has been included in [v1.71.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
